### PR TITLE
Rename Coveralls.submit_token to submit_local

### PR DESCRIPTION
**Pull request by fingolfin**
This matches the Codecov module

### CODE REVIEW

File: src/coveralls.jl [hunk 1]
```diff
@@ -185,4 +185,7 @@ module Coveralls
             println(String(r.body))
         end
     end
+
+    @deprecate submit_token submit_local
```
Comment (ararslan) [inline src/coveralls.jl]: This will need to be `Base.@deprecate`, assuming there isn't already a `using Base: @deprecate` in this module.

Comment (fingolfin) [reply src/coveralls.jl]: OK... So why isn't that needed in src/codecovio.jl (where I took this from)?

**Comment by fingolfin** [comment]
Travis completed, but still is shown as pending here; seems to be a glitch? Perhaps the Travis job needs to be restarted.

File: src/coveralls.jl (hunk 1 above)
Comment (ararslan) [reply src/coveralls.jl]: Whoa, weird... it looks like at some point `@deprecate` got exported from Base. Not sure when that happened. Never mind then.

File: src/coveralls.jl [hunk 2]
```diff
@@ -156,14 +156,15 @@ module Coveralls
...
-        submit_token(fcs::Vector{FileCoverage}, git_info=query_git_info; kwargs...)
+        submit_local(fcs::Vector{FileCoverage}, git_info=query_git_info; kwargs...)
 
     Take a `Vector` of file coverage results (produced by `process_folder`),
     and submits them to Coveralls. For submissions not from TravisCI.
 
     git_info can be either a `Dict` or a function that returns a `Dict`.
     """
-    function submit_token(fcs::Vector{FileCoverage}, git_info=query_git_info; kwargs...)
+
```
Comment (ararslan) [inline src/coveralls.jl]: This line break should be removed, otherwise the docstring doesn't apply to the function definition below it.

Comment (simonbyrne) [reply src/coveralls.jl]: I think that was due to my rebase.

**Pull request merged**